### ISSUE TITLE
Broken maximize function

### ISSUE THREAD

**Issue by ethernidee** [Flyer53/jsPanel4#141]
Hi! Thank you very much for the library.
Window maximize function does not work well. It should always set Top to 0, but instead it set it to some garbage value if page is scrolled first.

To test I open simple dialog on button click
```
<div class="ui button wnd-opener"">OPEN</div>

  <script type="text/javascript">
    jQuery(function ($) {     
      $('.wnd-opener').click(function (e) {
        e.preventDefault();

        jsPanel.create({
          contentAjax: "/someUrl",
        });
      });
    });
  </script>
```

After that scroll window 2-3 screens down (for full HD it's about 1080 * 2-3 pixels). And then press "maximize".
JsPanel window disappears at all with top: 2417px.

**Comment by Flyer53** [comment]
Hi there,

And thanks for the report. I can confirm there is an issue.

It seems this happens only when **option** `position` is not set in the panel config. At least in my first quick tests I could not find anything wrong when **option** `position` is set. 

Could you please check that in your app and tell me the result?

Thanks,
Stefan

**Comment by Flyer53** [comment]
**Addition:**
It looks like this issue appears whenever **option** `position` is set with an _object_.
Using a _string_ as value for **option** `position` seems to work fine.

**Comment by ethernidee** [comment]
Right, the issue occurs when no position is provided in options. Maybe you don't initialize "position" structure in this case. I didn't look deeply in the code.

**Comment by Flyer53** [comment]
I located the issue in the code and already have the fix ... I think.
It's not fully tested yet. But you can download and test a [preliminary version 4.11.1](https://alpha.jspanel.de/downloads/jspanel-4.11.1.zip).
I obviously forgot to remove a few lines of code for a positioning parameter that's not supported anymore.

I would appreciate if you gave it a ty and tell me the result....

**Comment by ethernidee** [comment]
It works now, Thank you very much!

**Comment by Flyer53** [comment]
That was fast ... thanks a lot 👍 
I'll have to do some testing of course before I release a new version.

**Comment by Flyer53** [comment]
Fixed with v4.11.1
